sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Authoring/ConversationAuthoringClient.cs

```diff
@@ -16,32 +14,31 @@ namespace Azure.AI.Language.Conversations.Authoring
     [CodeGenClient("ConversationalAnalysisAuthoringClient")]
     public partial class ConversationAuthoringClient
     {
-        /// <summary>
-        /// Initializes a new instance of ConversationAuthoringClient.
-        /// </summary>
+        /// <summary> Initializes a new instance of ConversationAuthoringClient. </summary>
         /// <param name="endpoint"> Supported Cognitive Services endpoint (e.g., https://&lt;resource-name&gt;.cognitiveservices.azure.com). </param>
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
-        public ConversationAuthoringClient(Uri endpoint, AzureKeyCredential credential) : this(endpoint, credential, new ConversationAnalysisClientOptions())
+        public ConversationAuthoringClient(Uri endpoint, AzureKeyCredential credential) : this(endpoint, credential, new ConversationsClientOptions())
         {
         }
 
-        /// <summary>
-        /// Initializes a new instance of ConversationAuthoringClient.
-        /// </summary>
+        /// <summary> Initializes a new instance of ConversationAuthoringClient. </summary>
         /// <param name="endpoint"> Supported Cognitive Services endpoint (e.g., https://&lt;resource-name&gt;.cognitiveservices.azure.com). </param>
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <param name="options"> The options for configuring the client. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
-        public ConversationAuthoringClient(Uri endpoint, AzureKeyCredential credential, ConversationAnalysisClientOptions options)
+        public ConversationAuthoringClient(Uri endpoint, AzureKeyCredential credential, ConversationsClientOptions options)
         {
             Argument.AssertNotNull(endpoint, nameof(endpoint));
             Argument.AssertNotNull(credential, nameof(credential));
-            options ??= new ConversationAnalysisClientOptions();
+            options ??= new ConversationsClientOptions();
 
             ClientDiagnostics = new ClientDiagnostics(options, true);
-            _keyCredential = credential;
-            _pipeline = HttpPipelineBuilder.Build(options, Array.Empty<HttpPipelinePolicy>(), new HttpPipelinePolicy[] { new AzureKeyCredentialPolicy(_keyCredential, AuthorizationHeader) }, new ResponseClassifier());
+
+            // BUGBUG: https://github.com/Azure/azure-sdk-for-net/issues/29506
```

Comment:
is BUGBUG one step above BUG? 😆

Reply:
Old school syntax since "BUG" (case-insensitively) was often too prevalent. IMO, "TODO"s are great for things like "implement this later", etc. This more: this is bad for reasons: {link}.